x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartTrainedModelDeploymentAction.java

```diff
@@ -244,6 +252,22 @@ public void setPriority(String priority) {
             this.priority = Priority.fromString(priority);
         }
 
+        public long getPerDeploymentMemoryBytes() {
```

Comment:
I can't see where these new fields added to the `StartTrainedModelDeploymentAction.Request` object are used

Reply:
There are 2 classes in the file that have a `getPerDeploymentMemoryBytes()` function. `StartTrainedModelDeploymentAction.Request` on line 255 and `StartTrainedModelDeploymentAction.TaskParams` on line 724.  The `TaskParams functions are used but I don't think the `Request` functions are

Reply:
Indeed, you are right. I ended up removing the fields from the `Request` class.